R/helper_item_deletion.R

```diff
@@ -0,0 +1,341 @@
+#' @title
+#' Compute PS, SR, SE, SP weighted by group proportions
+#' 
```

Comment:
It's a minor thing, but you can strip out the white spaces automatically if you use VS Code (Maybe RStudio as well.)